docs/install_guides/installing-openvino-apt.md

```diff
@@ -4,9 +4,15 @@
 
 This guide provides detailed steps for installing OpenVINO™ Runtime through the APT repository and guidelines for installing OpenVINO Development Tools.
 
-.. note:: From the 2022.1 release, OpenVINO™ Development Tools can be installed via PyPI only. See :ref:`Install OpenVINO Development Tools <installing-openvino-development-tools>` for more information.
+Installing OpenVINO Runtime from APT is recommended for C++ developers. If you are working with Python, the PyPI package has everything needed for Python development and deployment on CPU and GPUs. Visit the [Install OpenVINO from PyPI](installing-openvino-pip.md) page for instructions on how to install OpenVINO Runtime for Python using PyPI.
```

Comment:
I see that link `Visit the [Install OpenVINO from PyPI](installing-openvino-pip.md)` is not rendered correctly.

Reply:
can we align it with rpm installation ? they are totally similar in terms what they provide

Reply:
Good catch, thanks! Updated YUM installation as well. :)